breba_app/orchestrator.py

```diff
@@ -1,5 +1,6 @@
 import difflib
 import logging
+import chainlit as cl
```

Comment:
Orchestrator is part agent orchestration and should not have any knowledge of chainlit. Chainlit is a frontend provider and "my_cl_app" is the bridge between the front end and backend. my_cl app should be the only place talking to Chainlit